travis/script/functions.sh

```diff
@@ -83,6 +83,50 @@ function run_spec_suite_for {
   fi;
 }
 
+function check_binstubs {
+  echo "Checking required binstubs"
+
+  local success=0
+  local binstubs=""
+  local gems=""
+
+  if [ ! -x ./bin/rspec ]; then
+    binstubs=" bin/rspec"
+    gems=" rspec-core"
+    success=1
+  fi
+
+  if [ ! -x ./bin/rake ]; then
+    binstubs="$binstubs bin/rake"
+    gems="$gems rake"
+    success=1
+  fi
+
+  if [ ! -x ./bin/cucumber ]; then
+    binstubs="$binstubs bin/cucumber"
+    gems="$gems cucumber"
+    success=1
+  fi
+
+  if [ $success -eq 1 ]; then
+    echo
+    echo "Missing binstubs:$binstubs"
+    echo "Install missing binstubs using one of the following:"
+    echo
+    echo "  # Create the missing binstubs"
+    echo "  $ bundle binstubs$gems"
+    echo
+    echo "  # To binstub all gems"
+    echo "  $ bundle install --binstubs"
+    echo
+    echo "  # To binstub all gems and avoid loading bundler"
+    echo "  $ bundle install --standalone --binstubs"
+  fi
+
+  return $success
+}
+
+check_binstubs
```

Comment:
Why are you calling this here?  The check below in `run_build` makes sense but here it seems odd.